client/a8c-for-agencies/components/add-new-site-button/import-from-wpcom-modal/style.scss

```diff
@@ -0,0 +1,113 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.import-from-wpcom-modal {
+	max-width: 827px;
+	max-height: 100%;
+	margin: 0;
+
+	.components-modal__content {
+		padding: 0;
+
+		> div {
+			display: flex;
+			flex-direction: column;
+			justify-content: space-between;
+			height: 100%;
+			position: relative;
+		}
+	}
+
+	@include break-medium {
+		margin: auto;
+	}
+}
+
+.import-from-wpcom-modal__main {
+	padding: 40px;
+	gap: 8px;
+	display: flex;
+	flex-direction: column;
+	height: 680px;
+
+	@include break-large {
+		width: 800px;
+	}
+}
+
+.import-from-wpcom-modal__title {
+	font-size: rem(20px);
+	font-weight: 600;
+	line-height: 1.2;
+	margin: 0;
+	padding: 0;
+}
+
+p.import-from-wpcom-modal__instruction {
+	font-size: rem(12px);
```

Comment:
We can use the A4A typography mixin here to set font properties.
```
@include a4a-font-body-sm;
```

Reply:
Ah, yes, I thought the PR wasn't merged. Thanks! I have updated now: https://github.com/Automattic/wp-calypso/pull/92313/commits/1e0c630b1b6a8a967e6017b1c65710fa2974b65b